etc/workflows/failedRecording.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>failedrecording</id>
+  <title>Recording failed</title>
```

Comment:
Imagine a user seeing "Recording failed" in the workflow selection dropdown. Will she or he understand what this is supposed to do? 

Maybe something more descriptive like "Publish failed placeholder"...

Reply:
done